test/src/sender/main.cpp

```diff
@@ -13,129 +13,151 @@
- "TOP", 
- submodels_TOP, 
- iports_TOP, 
- oports_TOP, 
- eics_TOP, 
- eocs_TOP, 
- ics_TOP 
-  );
-
-///****************////
-
-    auto elapsed1 = std::chrono::duration_cast<std::chrono::duration<double, std::ratio<1>>>(hclock::now() - start).count();
+    using info = cadmium::logger::logger <cadmium::logger::logger_info, 
+															cadmium::dynamic::logger::formatter <TIME>, oss_sink_provider> ;
+    using debug = cadmium::logger::logger <cadmium::logger::logger_debug, 
+															cadmium::dynamic::logger::formatter <TIME>, oss_sink_provider> ;
+    using state = cadmium::logger::logger <cadmium::logger::logger_state, 
+															cadmium::dynamic::logger::formatter <TIME>, oss_sink_provider> ;
+    using log_messages = cadmium::logger::logger <cadmium::logger::logger_messages,
+															cadmium::dynamic::logger::formatter <TIME>, oss_sink_provider> ;
+    using routing = cadmium::logger::logger 
+									<cadmium::logger::logger_message_routing, cadmium::dynamic::logger::formatter
+																									<TIME>, oss_sink_provider> ;
+    using global_time = cadmium::logger::logger 
+									<cadmium::logger::logger_global_time, cadmium::dynamic::logger::formatter 
+																									<TIME>, oss_sink_provider> ;
+    using local_time = cadmium::logger::logger <cadmium::logger::logger_local_time,
+																		cadmium::dynamic::logger::formatter <TIME>, oss_sink_provider> ;
+    using log_all = cadmium::logger::multilogger <info, debug, state,
+															log_messages, routing, global_time, local_time> ;
+    using logger_top = cadmium::logger::multilogger <log_messages, global_time> ;
+
+    /*******************************************/
+
+    /********************************************/
+    /****** APPLICATION GENERATOR *******************/
+    /********************************************/
+    string input_data_control = "test/data/sender_input_test_control_In.txt"; 
+	const char *i_input_data_control = input_data_control.c_str();
+
+    std::shared_ptr <cadmium::dynamic::modeling::model> generator_con = 
+															cadmium::dynamic::translate::make_dynamic_atomic_model 
+																										< ApplicationGen, TIME, const char *> ("generator_con", 
+																																				std::move(i_input_data_control));
+
+    string input_data_ack = "test/data/sender_input_test_ack_In.txt"; 
+	const char *i_input_data_ack = input_data_ack.c_str();
+
+    std::shared_ptr <cadmium::dynamic::modeling::model> generator_ack = 
+															cadmium::dynamic::translate::make_dynamic_atomic_model 
+																										< ApplicationGen, TIME, const char *> ("generator_ack", std::move 
+																																								(i_input_data_ack));
+
+    /********************************************/
+    /****** SENDER *******************/
+    /********************************************/
+
+    std::shared_ptr <cadmium::dynamic::modeling::model> sender1 = 
```

Comment:
alignment issue. when you fix the code, please see if it looks more readable in your laptop screen and then submit for review.